### ISSUE TITLE
All node entities on the same position.

### ISSUE THREAD

**Issue by bell07** [stujones11/meshnode#11]
In some cases the ship is unusable broken because ship-node-entities are at the same position. It looks  like one node but with textures of all materials visible. I observed the issue 2x times but did not taken the screenshots. As I tried it again to take the screenies the issue did not appears anymore, so it is any special constellation.

As the issue appears:
1. I stopped the server during my player was attached to flying ship. After restart I was not in game but an second player looked to the ship that was broken.

2. I leaved my ship, goes away till the enitiy was unloaded, then came back and my ship was not useable.

Any idea what is happened?

**Comment by stujones11** [comment]
Strange, I've not actually tried this in a while so maybe something changed in MT, will take a look as soon as I get a chance. If you can describe a simple repeatable method to reproduce the problem it would be very helpful, as would screenshots if you see it again.

**Comment by bell07** [comment]
Now I was able to reproduce the issue. minetest_game + meshnode mods only in creative:
1 create a small ship with dirt_with_gras and enable them.
2 create a second small ship with dirt_with_gras and enable them too.
3 go away till entities disabled
4. come back

Result:
![screenshot_20180110_205246](https://user-images.githubusercontent.com/4731334/34792497-793db6d8-f648-11e7-84bb-a9ff5b94b9ec.png)

**Comment by stujones11** [comment]
I have followed this procedure as closely as possible but have not yet been able to reproduce the problem. I can only guess that the controller's 'staticdata' is being corrupted somehow. Clearly there is a problem somewhere, I will continue to test.

Do you see any unusual warnings about stray objects or serialization errors? Also, which MT version is this?

**Comment by bell07** [comment]
Second try, now it was more difficult to reproduce the issue. This time I was able to see the issue after I created and activated the both meshes, stopped the server, started again with a second player (I use the Minetest GUI with "Host Server" flag)

On console I see nothing related. Does the [debug.txt.gz](https://github.com/stujones11/meshnode/files/1636867/debug.txt.gz) help?

**Comment by stujones11** [comment]
Yeah, I only tried with singleplayer but will try with local multiplayer when I get a chance. I cannot see anything in the logs either so it's likely going to be a difficult one to trace.

I think that bottom line here is that entity static data is simply not reliable enough for this task. I may look at converting it to use the mod storage api instead.
